bin/yaml/julia.yaml

```diff
@@ -21,3 +22,7 @@ compilers:
           url: https://julialangnightlies-s3.julialang.org/bin/linux/x86_64/julia-latest-linux-x86_64.tar.gz
           strip_components: 1
           create_untar_dir: true
+    depot: {destination}/depot
```

Comment:
I have no clue what path `destination` is, I saw it in https://github.com/compiler-explorer/infra/blob/f7d9fb5ecdc900be52a8a4664338800d7cf1760e/bin/yaml/haskell.yaml#L14 but I don't know if `{destination}/depot` is something like `/opt/compiler-explorer/julia-{name}` or what.

Reply:
You may need to quote some of these things to make them valid `yaml`